app/controllers/games_controller.rb

```diff
@@ -71,17 +70,14 @@ def game_subset
   end
 
   def search(subset)
-    if params[:search_user]
-      subset.search_by('user', params[:search_user])
-    elsif params[:tag]
-      subset.search_by('tags', params.require(:tag)[:tag_ids])
-    elsif params[:search_main]
-      subset.search_by('main', params[:search_main])
-    elsif params[:search_state]
-      subset.search_by('state', params[:search_state])
-    else
-      subset
+    results = subset
```

Comment:
the name `subset` is confusing here. Reading `results = subset` doesn't tell me much, maybe find a better name to the `subset` argument